README.md

```diff
@@ -7,51 +7,57 @@ A Swift mixin to use `UITableViewCells`, `UICollectionViewCells` and `UIViewCont
 [![Platform](http://cocoapod-badges.herokuapp.com/p/Reusable/badge.png)](http://cocoadocs.org/docsets/Reusable)
```

Comment:
fyi `<center>![alt](url)</center>` doesn't seem to work. Had to rely on `<p align="center">` + using an `<img>` tag instead of markdown :wink: